mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -764,13 +764,17 @@ class LinearRegressionSuite
           (Intercept) 6.3022157  0.0018600    3388   <2e-16 ***
           V2          4.6982442  0.0011805    3980   <2e-16 ***
           V3          7.1994344  0.0009044    7961   <2e-16 ***
+
+          # R code for r2adj
+          summary(lm(X1 ~ ., data = part_00000))
```

Comment:
this still doesn't test the without intercept case. I think we actually need a test just to test the summary, where we can test with/without intercept r2 and other metrics against R code. Would you mind making another standalone test that compares the linear regression summary to R (both with/without intercept)?

Reply:
The existing test "Linear regression model training summary" does not test without intercept case. If there is a "Linear regression model training summary wo intercept", I am glad to add in it.